.github/workflows/helm-chart-release.yml

```
@@ -90,6 +94,38 @@ jobs:
           keep_files: true
           enable_jekyll: true
 
+      - name: Upload image file
+        shell: bash
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          set -e
+          # .cr-release-packages is the directory used by the Helm releaser from a previous step
+          chart_directory=.cr-release-packages
+          if [ ! -d "$chart_directory" ]; then
+            echo "$chart_directory does not exist. Assuming no charts update"
+            exit 0
+          fi
+
+          charts=$(find ./charts -maxdepth 1 -mindepth 1 -type d)
+          asset_name=""
+          for chart in $charts; do
+            chart_name=$(helm show chart $chart | yq -r '.name' )
+            chart_version=$(helm show chart $chart | yq -r '.version')
+            asset_name="${asset_name}_${chart_name}-${chart_version}"
+          done
+          asset_name="${asset_name:1}_images.txt"
+          mv imagelist.txt $asset_name
```

```suggestion
          cp imagelist.txt $asset_name
```

just in case we reorder things or we are debugging.